File: main-2.py
def convert(n):

    result = []
# looping on string
    for num in str(n):
        result.insert(0, int(num))

    return result


print(convert(564987654))  # [4, 5, 6, 7, 8, 9, 4, 6, 5]
print(convert(529132))  # [2, 3, 1, 9, 2, 5]

File: main2.js
function convert(n) {
  return n.toString().split("").map((m) => +m);
}

console.log(convert(564987654)); // [4, 5, 6, 7, 8, 9, 4, 6, 5]
console.log(convert(529132)); // [2, 3, 1, 9, 2, 5]
